config/gradle/publish.gradle

```diff
@@ -17,16 +17,45 @@ publishing {
                 maven {
                     name = 'TerasologyOrg'
 
-                    def repoViaEnv = System.getenv()["PUBLISH_REPO"]
                     if (rootProject.hasProperty("publishRepo")) {
```

Comment:
I'm wondering whether we should start to scope the properties in the gradle builds, similar to what most plugins to. This would become `terasology.publishRepo` then...

Reply:
Had not thought about that before. Even in a plain build file, rather than a plugin? I've been curious if some of our logic might go better _in_ a plugin, where it would make all kinds of sense, since then you'd probably use neato little blocks to configure it.